cmd/validate/validateNavigation.go

```diff
@@ -15,6 +18,18 @@ func validateNavigation(cwd string) {
 	for _, file := range modulesFiles {
 		documentLoader := gojsonschema.NewReferenceLoader(fmt.Sprintf("file://%s", file))
 		result, err := gojsonschema.Validate(schemaLoader, documentLoader)
+		
```

Comment:
We should not skip the error check. Each block of code should have an escape hatch if it errors.